### PR TITLE
es6-ify code (use classes, let, import)

### PR DESCRIPTION
**Pull request by avindra**
I changed some of the requires to import statements, replaced var with let or const, used native class syntax instead of React.createClass, etc.

I've only done conversions in places where it seemed safe, I can do the rest with further testing.

### CODE REVIEW

**Comment by brentvatne** [comment]
Looks good to me, @skv-headless?

File: index.js [hunk 1]
```diff
@@ -11,13 +10,28 @@ var {
...
 
-var DefaultTabBar = require('./DefaultTabBar');
-var deviceWidth = Dimensions.get('window').width;
-var deviceHeight = Dimensions.get('window').height;
+const DefaultTabBar = require('./DefaultTabBar');
+const deviceWidth = Dimensions.get('window').width;
+const deviceHeight = Dimensions.get('window').height;
 
-var ScrollableTabView = React.createClass({
+const styles = StyleSheet.create({
```
Comment (skv-headless) [inline index.js]: Why not on the bottom?

**Comment by skv-headless** [comment]
I see that every library use their own convention. In our project we use
1) ```const``` instead of ```var``` and ```let```. almost everywhere.
2) import over require
3) https://github.com/Khan/style-guides/blob/master/style/react.md for component

Do you guys know advantages/disadvantages for such approach?
I hope in some day all libraries will be in the same style.

**Comment by skv-headless** [comment]
also es6 classes has drawbacks.
1) no way to use mixins
2) no automatic function binding to component

File: DefaultTabBar.js [hunk 2]
```diff
@@ -42,9 +41,9 @@ var DefaultTabBar = React.createClass({
   },
 
   renderTabOption(name, page) {
-    var isTabActive = this.props.activeTab === page;
-    var activeTextColor = this.props.activeTextColor || "navy";
-    var inactiveTextColor = this.props.inactiveTextColor || "black";
+    let isTabActive = this.props.activeTab === page;
```
Comment (skv-headless) [inline DefaultTabBar.js]: please replace all ```let``` with ```const``` if possible

File: examples/FacebookTabsExample/FacebookTabsExample.js [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
+  Component,
+} from 'react-native';
 
-var ScrollableTabView = require('react-native-scrollable-tab-view');
-var FacebookTabBar = require('./FacebookTabBar');
+const ScrollableTabView = require('react-native-scrollable-tab-view');
+const FacebookTabBar = require('./FacebookTabBar');
 
-var FacebookTabsExample = React.createClass({
+class FacebookTabsExample extends Component {
```
Comment (skv-headless) [inline examples/FacebookTabsExample/FacebookTabsExample.js]: I would prefer ```React.createClass```

**Pull request closed without merging**